ci/ansible/roles/pulp_certs/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Create CA to sign all certificates
+  command: >
+    /usr/bin/openssl req -x509 -newkey rsa:2048 -keyout private/cakey.pem -nodes -days 3650 \
```

Comment:
By using `>` YAML parser is going to add everything in one line converting each `\n` to a single space. I did not used `\` to separate lines on my recent PR (https://github.com/pulp/pulp_packaging/pull/105/files#diff-0760f8ccddb91707cb83e68f060aee9eR9) and it worked.

I'm just comment to let you know it, feel free to change if you prefer dropping the `\`.